src/blog/2024/08/comparing-dashboard-2-with-uibuilder.md

```diff
@@ -0,0 +1,233 @@
+---
+title: "Node-RED Dashboard 2.0 vs UI-Builder: A Comprehensive Comparison"
+subtitle: Understanding the Differences Between Node-RED Dashboard 2.0 and UI-Builder
+description: Compare Node-RED Dashboard 2.0 and UI-Builder. Discover their installation ease, customization, performance, and support to find the best solution for your needs. 
+date: 2024-08-31
+authors: ["sumit-shinde"]
+image: 
+tags:
+    - node-red dashboard
+    - ui-builder
+    - comparison
+    - dashboard
+
+---
+
+When choosing a dashboard solution for Node-RED, two popular options are Node-RED Dashboard 2.0 and UI-Builder. This article compares these tools across several key areas, including installation, ease of use, development activity, and customizability, to help you decide which one best suits your needs.
+
+<!--more-->
+
+## How Easy is it to Install?
```

Comment:
Would be nice to have a single image split with left-side being a Dashboard 2.0 screenshot and right-side being a UI Builder screenshot. Similar to waht we have in https://flowfuse.com/blog/2023/03/comparing-node-red-dashboards/

This could also just be the blog tile/image once it's ready